Library/Homebrew/livecheck.rb

```diff
@@ -44,7 +45,13 @@ def skip?
   def url(val = nil)
     return @url if val.nil?
 
-    @url = val
+    @url = if (val.eql? :head) || (val.eql? :stable)
```

Comment:
Should probably support `:devel` here too. Also, I'd suggest using a `case` statement.